ut/s3_bucket_metadata_v1_test.cc

```diff
@@ -568,29 +568,6 @@ TEST_F(S3BucketMetadataV1Test, SaveBucketInfo) {
       action_under_test->system_defined_attribute["Owner-Account-id"].c_str());
 }
 
-TEST_F(S3BucketMetadataV1Test, SaveBucketInfoSuccess) {
```

Comment:
why this test is removed ?

Reply:
modify this test according to the latest change.

Reply:
save_replica() and remove_replica() functions have been removed so their UTs are removed. 
for save_bucket_info_successful() and remove_bucket_info_successful(), UT's are fixed.
Please have a look.